packages/@aws-cdk/core/test/bundling.test.ts

```diff
@@ -438,6 +438,43 @@ describe('bundling', () => {
     ], { stdio: ['ignore', process.stderr, 'inherit'] })).toEqual(true);
   });
 
+  test('adding user provided docker volume options', () => {
+    // GIVEN
+    sinon.stub(process, 'platform').value('darwin');
+    const spawnSyncStub = sinon.stub(child_process, 'spawnSync').returns({
+      status: 1,
+      stderr: Buffer.from('stderr'),
+      stdout: Buffer.from('stdout'),
+      pid: 123,
+      output: ['stdout', 'stderr'],
+      signal: null,
+    });
+    const image = DockerImage.fromRegistry('alpine');
+
+    try {
+      image.run({
+        command: ['cool', 'command'],
+        volumesFrom: ['foo'],
```

Comment:
For arrays I usually like to see 2+ entries 😅 
```suggestion
        volumesFrom: ['foo', 'bar'],
```